contrib/fine-tuning/fine-tune-mlx.sh

```diff
@@ -13,7 +16,7 @@ case "$TOOL_BASE_MODEL" in
 esac
 BASE_MODEL_MLX=${BASE_MODEL}-${TUNING_TOOL}
 HF_ORG=CycloneDX
-NUM_LAYERS=16
+NUM_LAYERS=32
```

Comment:
While working on cdx1-mini model, I noticed the bug with our lora fine-tuning. The performance for cdx1 and cdx1-pro can be improved slightly in a future iteration.